src/screens/GetStartedScreen.tsx

```diff
@@ -0,0 +1,28 @@
+import { View, Text, TouchableOpacity } from 'react-native';
+import React from 'react';
+import { useNavigation } from '@react-navigation/native';
+
+const GetStarted = () => {
+  const navigation = useNavigation();
```

Comment:
I think it's fine this way, but in other screen, especcialy to which we're passing props, it'd be betteer to use navigation prop
`const GetStarted = ({navigation}: NativeStackScreenProps<RootStackParamList, rootScreens.GetStarted>) => {...`

`useNavigation` is helpfull when we need to use navigation from inside a component, but it'd be better to create a custom hook that's typed to use our screens